features/steps/partcad-cli/commands/add/assembly.py

```
@@ -0,0 +1,36 @@
+from behave import given, when, then
+from hamcrest import assert_that, equal_to
+from features.utils import expandvars  # type: ignore # TODO: @alexanderilyin python.autoComplete.extraPaths
+import logging
+import yaml
+import os
+
+
+@given('a file named "{filename}" with content')
+def step_impl(context, filename):
```

Ruff (0.8.0)

Missing return type annotation for public function `step_impl`

Add return type annotation: `None`

(ANN201)